src/index.js

```diff
@@ -8,11 +8,19 @@ import PostData from './post-data';
 
 import './nhsuk-feedback-form.css';
 
+function getEnableTextResponse(container) {
+  if (container.getAttribute('data-enable-text-response') === 'true') {
```

Comment:
You'll probably end up changing this due to the boolean attribute thing, but it could have been a one-line function instead.
```js
function getEnableTextResponse(container) {
  return container.getAttribute('data-enable-text-response') === 'true';
}
```

Reply:
Taken your recommendation. Instead of using "getAttribute" I've used "hasAttribute" to avoid using true/false in the data attribute.